### PR TITLE
docs: Improve Documentation for JWT/JWK

### PR DESCRIPTION
**Pull request by ovr**
Hello!

Thanks

### CODE REVIEW

File: docs/content/Authentication-&-Authorization.md [hunk 1]
```diff
@@ -270,3 +270,90 @@ module.exports = {
...
+## Auth0
+
+<!-- prettier-ignore-start -->
+[[warning | Note]]
+| There is an [example/auth](https://github.com/cube-js/cube.js/tree/master/examples/auth0) inside Cube.js repository.
+<!-- prettier-ignore-end --
+
+1. Navigate to the [Auth0 dashboard](https://manage.auth0.com/).
+2. Click on the APIs menu option on the left sidebar and then click the Create API button.
+3. Fill the name, fill the Identifier (we recommend to use URL), select Signing Algorithm (We highly recommend to use `RS256`).
```
Comment (rpaik) [inline docs/content/Authentication-&-Authorization.md:283]:
```suggestion
3. Fill in the name, fill in the Identifier (we recommend using URL), and select Signing Algorithm (We highly recommend using `RS256`).
```

File: docs/content/Authentication-&-Authorization.md [hunk 2]
```diff
@@ -270,3 +270,90 @@ module.exports = {
...
+
+1. Create App
+2. Fill Allowed Callback Urls, Allowed Web Origins, Allowed Origins (CORS)
```
Comment (rpaik) [inline docs/content/Authentication-&-Authorization.md:286]:
```suggestion
2. Fill in Allowed Callback Urls, Allowed Web Origins, Allowed Origins (CORS)
```

File: docs/content/Authentication-&-Authorization.md [hunk 3]
```diff
@@ -270,3 +270,90 @@ module.exports = {
...
+CUBEJS_JWT_ISSUER=https://ovr.auth0.com/
+# Supported algorithm of your Custom API
+CUBEJS_JWT_ALG=RS256
+# You can copy/past JWK url from your API settings.
+CUBEJS_JWK_URL=https://ovr.auth0.com/.well-known/jwks.json
+```
+
+### Custom Token Claims
+
+By default, there are no claims inside JWT's payload, but it's possible to do by creating custom rule.
```
Comment (rpaik) [inline docs/content/Authentication-&-Authorization.md:315]:
```suggestion
By default, there are no claims inside JWT's payload, but it's possible by creating a custom rule.
```

File: docs/content/Authentication-&-Authorization.md [hunk 4]
```diff
@@ -270,3 +270,90 @@ module.exports = {
...
+
+<!-- prettier-ignore-start -->
+[[warning | Note]]
+| There is an [example/cognito](https://github.com/cube-js/cube.js/tree/master/examples/cognito) inside Cube.js repository.
+<!-- prettier-ignore-end --
+
+### Custom Token Claims
+
+By default, there are no claims inside `accessToken`'s token payload, but it's possible to be done by lamda on top of
+Pre-Token Generation trigger.
```
Comment (rpaik) [inline docs/content/Authentication-&-Authorization.md:349]:
```suggestion
the Pre-Token Generation trigger.
```

**Comment by hassankhan** [comment]
@ovr @keydunov Is this PR still valid?

**Comment by keydunov** [comment]
It seems obsolete, let's close it.

**Pull request closed without merging**